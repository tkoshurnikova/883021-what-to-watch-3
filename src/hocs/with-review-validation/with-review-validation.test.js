import React from "react";
import renderer from "react-test-renderer";
import withReviewValidation from "./with-review-validation.jsx";

const MockComponent = () => <div />;
const MockComponentWrapped = withReviewValidation(MockComponent);

it(`Render withReviewValidation`, () => {
  const tree = renderer
    .create((
      <MockComponentWrapped/>),
    {
      createNodeMock() {
        return {};
      }
    }).toJSON();

  expect(tree).toMatchSnapshot();
});
