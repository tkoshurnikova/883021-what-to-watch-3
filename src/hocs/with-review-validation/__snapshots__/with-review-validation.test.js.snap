// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render withReviewValidation 1`] = `<div />`;
